webodf/lib/core/Zip.js

```diff
@@ -168,7 +168,7 @@ core.Zip = function Zip(url, entriesReadCallback) {
      */
     function createByteArray(successCallback, errorCallback) {
         try {
-            successCallback(/**@type{!Uint8Array}*/(zip.generate({type: "uint8array", compression: "DEFLATE"})));
+            successCallback(/**@type{!Uint8Array}*/(zip.generate({type: "uint8array", compression: "STORE"})));
```

Comment:
If I'm reading the JSZip API correctly, you theoretically don't need both of these specified. It appears that the ``generate`` compression is a fallback if no explicit level is set on the ``file``. Does that match your understanding?

If that is the case, I'd be interested as to why or how the existing code was compressing the mimetype.